.github/workflows/bump-cs-version.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         run: |
           ./.ci/bump-cs-version-in-sevntu-checks.sh ${{ steps.VERSION.outputs.LATEST_VERSION }}
           ./.ci/bump-cs-version-in-sonar-plugin.sh ${{ steps.VERSION.outputs.LATEST_VERSION }}
+          ./.ci/bump-cs-version-in-eclipsecs-sevntu-plugin.sh \
```

Comment:
Please remove 'sevntu' from name, I hope it will give room to make it single line

Reply:
It does.

Reply:
Done.